android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -1156,10 +1173,15 @@ public void setBufferConfig(int newMinBufferMs, int newMaxBufferMs, int newBuffe
      * @param controls  Controls prop, if true enable controls, if false disable them
      */
     public void setControls(boolean controls) {
-        if (controls && exoPlayerView != null) {
+        this.controls = controls;
```

Comment:
This appears to be an unused variable.

Reply:
It is being used there:
https://github.com/react-native-community/react-native-video/pull/1655/files/40c7371b3e8955ffae516dbda1a8783952c14381#diff-2b2f68a93f154d6095529ae77eeb90f1R395

in this line `this` is explicit because argument is named the same